sql/moz-fx-data-marketing-prod/acoustic/contact_v1/init.sql

```diff
@@ -1,48 +0,0 @@
-CREATE TABLE IF NOT EXISTS
```

Comment:
Out of curiosity, what is the purpose of an init.sql file in bqetl and why are you deleting them here?